python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -517,3 +495,45 @@ def _private_display(self, reg_slice, this_format, default_width, markup):
             ds += markup(this_format % tuple(vals), colour)
 
         return ds
+
+    @staticmethod
+    def JobHasChildrenTest(jobDirectory):
+        """ Return True/False if given (job?) object has children associated with it
+        Args:
+            jobDirectory (str): name of folder to be examined
+        """
+
+        count = SubJobXMLList.countSubJobDirs(jobDirectory)
+        if count:
+            return True
+        else:
+            return False
```

Comment:
(aside I hate casting to bools but c'est la vie)

I was planning to use/change this more in the future and to expand/change it. I'm still keen to keep a function which returns a True/False as it's much clearer what's going on.